### ISSUE TITLE
Enable / disable script in context menu based on object field values

### ISSUE THREAD

**Issue by mtrezza** [parse-community/parse-dashboard#2487]
### New Feature / Enhancement Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-dashboard/issues?q=is%3Aissue).

### Current Limitation

Currently, a script is visible in the data browser cell context menu based on the object class and ([soon](https://github.com/parse-community/parse-dashboard/issues/2485#issuecomment-1612106920)) object field.

### Feature / Enhancement Description

Enable / Disable script in context menu based on a combination of object field values. The script should be shown but grayed out in the context menu, so that the dashboard user knows that a script is available but currently disabled based on field values.

### Example Use Case

For example, a purchase transaction in which there is a buyingUser and sellingUser, but only the buyingUser can be flagged:

```json
"apps": [
  {
    "scripts": [
      {
        "title": "Flag fraudulent purchase",
        "classes": ["Transaction$buyingUser"],
        "requiredObjectValues": {
           "completed": true,
           "status": 1,
           "country": "US"
        },
        "cloudCodeFunction": "flagUser"
      }
    ]
  }
]
```

The conditions should support `bool`, `string`, `number` types and conditions should be `AND` related, i.e. all conditions need to be met for a script to be enabled. The default (if no conditions are set if that no fields are required and the script is always enabled.

There may be a better syntax for how to define these conditions, especially considering possible future additions (see below).

### Possible Future Extension

- Support combinations `AND`, `OR`
- Support comparative conditions like `<`, `>=`,  `!=`

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [6.0.0-alpha.8](https://github.com/parse-community/parse-dashboard/releases/tag/6.0.0-alpha.8)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [6.0.0-beta.2](https://github.com/parse-community/parse-dashboard/releases/tag/6.0.0-beta.2)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [6.0.0](https://github.com/parse-community/parse-dashboard/releases/tag/6.0.0)
